### ISSUE TITLE
word_classifier_CTC.ipynb accuracy+padding part

### ISSUE THREAD

**Issue by Lucifer-demon** [Breta01/handwriting-ocr#121]
@Breta01 
targets = tf.sparse_placeholder(dtype=tf.int32,
                                name='targets')

targets_length = targets.get_shape().as_list()[1]   it raises an error   the shpae of the tensor in unknown, we can't call as_list function
and more details like PAD , test_targets are not initialized.

**Comment by dubbsbrandon** [comment]
Did you happen to get this working? I'm at this same point and dont really want to use the Tensorboard portion, and just want the predictions.

**Comment by Lucifer-demon** [comment]
yeah, i have run the CTC file. Howerver, i delete the codes about accuracy. and the name of some variables are changed in the Tensorboard portion part but dont update in the CTC file. You can modify the codes according the error. If you have any problems, you can email me.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Brandon Dubbs"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月5日(星期四) 凌晨4:33
收件人:&nbsp;"Breta01/handwriting-ocr"<handwriting-ocr@noreply.github.com&gt;;
抄送:&nbsp;"545322747"<545322747@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [Breta01/handwriting-ocr] word_classifier_CTC.ipynb accuracy+padding part (#121)




Did you happen to get this working? I'm at this same point and dont really want to use the Tensorboard portion, and just want the predictions.

**Comment by dubbsbrandon** [comment]
Whats your contact information? I can't seem to get it to work. Do you have the word_classifier_CTC notebook working on your own dataset or...? I can't seem to get my own data to work.

**Comment by Lucifer-demon** [comment]
I use wechat more often. Your own data means the data you create or you download from the website the author provides? I download iam datasets, and if you use the data you create, i am afraid the format will be complex.



---Original---
